sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2023-2024.bpmn

```diff
@@ -808,6 +802,32 @@
       <bpmn:incoming>Flow_0lp9mvw</bpmn:incoming>
       <bpmn:outgoing>Flow_1pbkzeb</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
+    <bpmn:sequenceFlow id="Flow_0sp9tv0" sourceRef="Activity_1vwijyc" targetRef="Event_1mkiqso" />
+    <bpmn:scriptTask id="Activity_1vwijyc" name="Define Dependents">
+      <bpmn:extensionElements>
+        <zeebe:script expression="=// Consider only dependents who were born on or before the study end date.&#10;inputDataDependents[date(item.dateOfBirth) &#60;= date(offeringStudyEndDate)]" resultVariable="calculatedDataDependants" />
+        <zeebe:ioMapping>
+          <zeebe:input source="=//TODO: Student appeal variable needs to be considered here.&#10;if studentDataDependants = null &#10;then []&#10;else studentDataDependants" target="inputDataDependents" />
```

Comment:
Sorry, after approving I realized this: The script task should be after  "Define PD/PPD status" as we're going to the calculated variables for the appeals in parallel with that one.

Reply:
Besides that, I loved the idea of using the task and adjusting the inputs but for this one the appeals are supposed to be handled already, right?

Reply:
As we discussed, created this script task in a way that it is expected to be moved inside list of parallel tasks to be done for request a change.